app/src/main/res/values/strings.xml

```diff
@@ -692,4 +692,8 @@
     <string name="show_thumbnail_title">Show thumbnail</string>
     <string name="show_thumbnail_summary">Use thumbnail for both lock screen background and notifications</string>
     <string name="recent">Recent</string>
+    <string name="duplicate_video_in_playlist_title">Duplicated Video Found</string>
+    <string name="duplicate_video_in_playlist_description">This video had already added to this playlist %d times before, Do you want to add it one more time?</string>
```

Comment:
I suggest something like the string below. However, we need to keep in kind that `%d` can be 1+, thus some kind of `plurals` should be used to handle the different.
```suggestion
    <string name="duplicate_stream_in_playlist_description">The playlist contains this stream already %d time(s).\nDo you want to add it one more time?</string>
```